metar/Metar.py

```diff
@@ -286,43 +290,49 @@ def xlate_loc( loc ):
 
 ## Helper functions
 
-def _report_match(handler,match):
+def _report_match(handler, match):
   """Report success or failure of the given handler function. (DEBUG)"""
   if match:
-      print(handler.__name__," matched '"+match+"'")
+      _logger.debug("%s matched '%s'", handler.__name__, match)
   else:
-      print(handler.__name__," didn't match...")
-      
+      _logger.debug("%s didn't match...", handler.__name__)
+
+
 def _unparsedGroup( self, d ):
     """
     Handle otherwise unparseable main-body groups.
     """
     self._unparsed_groups.append(d['group'])
-      
+
 ## METAR report objects
 
 debug = False
```

Comment:
Seems like this *should* be a parameter in the Metar class. The question is if we want to make that a part of this change.